src/rest_api/classes/anatomy_term.clj

```diff
@@ -1,13 +1,16 @@
 (ns rest-api.classes.anatomy-term
   (:require
     [rest-api.classes.gene.widgets.external-links :as external-links]
-    ;[rest-api.classes.anatomy_term.widgets.overview :as overview]
-    [rest-api.classes.anatomy_term.widgets.ontology-browser :as ontology-browser]
+    ;[rest-api.classes.anatomy-term.widgets.overview :as overview]
+    [rest-api.classes.anatomy-term.widgets.references :as references]
+    [rest-api.classes.anatomy-term.widgets.ontology-browser :as ontology-browser]
     [rest-api.routing :as routing]))
 
 (routing/defroutes
-  {:entity-ns "anatomy-term"
+  {:entity-ns "anatomy-function"
```

Comment:
@a8wright I don't think this is correct. The Anatomy page on wb site definitely corresponds to anatomy term ace object. You can see it by going to an anatomy page and see the ace tree view: http://www.wormbase.org/species/all/anatomy_term/nerve%20ring%20neuron#014--10